apps/src/music/views/Generate.tsx

```diff
@@ -0,0 +1,195 @@
+import {Button} from '@code-dot-org/component-library/button';
+import React, {useCallback, useState} from 'react';
+
+import Adlib from '@cdo/apps/lab2/views/components/guide/Adlib';
+import Guide from '@cdo/apps/lab2/views/components/guide/Guide';
+import {useAppDispatch, useAppSelector} from '@cdo/apps/util/reduxHooks';
+
+import askAi from '../ai/generate/askAi';
+import {generateBlocklyJson} from '../ai/generate/generateBlocklyJson';
+import appConfig from '../appConfig';
+import MusicLibrary from '../player/MusicLibrary';
+import {setCodeToLoad, setAiGenerateState} from '../redux/musicRedux';
+
+import styles from './Generate.module.scss';
+
+const adlibs: {
```

Comment:
Similarly, pull this out into a separate JSON file while we figure out the right longer term solution?